csharp/build.sh

```diff
@@ -37,7 +37,7 @@ export CONFIGURATION=$STEP
 
 export STEP=$CONFIGURATION
 
-xbuild /p:Configuration=$CONFIGURATION $XBUILDOPT $PROJ
+xbuild /p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true $XBUILDOPT $PROJ
```

Comment:
Probably you can quote them just like : 
xbuild "/p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true"

Wait a minute, I'll restart my computer to verify that on virtual CentOS.